idx/cassandra/cassandra.go

```diff
@@ -394,15 +394,15 @@ func (c *CasIdx) load(defs []schema.MetricDefinition, iter cqlIterator, now time
 			orgId = int(idx.OrgIdPublic)
 		}
 
-		mdef := &schema.MetricDefinition{
+		mdef := &idx.MetricDefinition{
 			Id:         mkey,
 			OrgId:      uint32(orgId),
 			Partition:  partition,
-			Name:       name,
+			Name:       idx.MetricNameFromString(name),
 			Interval:   interval,
 			Unit:       unit,
```

Comment:
`Unit` and `Mtype` need to use the stringPool

Reply:
mtype only has a small set of possible values. (see MetricDefinition.Validate). better to just encode them as uint8 enums